# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ncurses/hotfix.patch.cross
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ncurses-6.2/misc/terminfo.src.vanilla	2020-02-17 19:32:57.344491480 +0000
+++ ncurses-6.2/misc/terminfo.src	2020-02-17 19:33:31.896491301 +0000
@@ -4782,17 +4782,6 @@
 
 #### XTERM Colors
 
-# 16-colors is one of the variants of XFree86 3.3 xterm, updated for 4.0
-# (T.Dickey)
-#
-# If configured to support 88- or 256-colors (which is fairly common in 2009),
-# xterm also recognizes the control sequences for initc -TD
-xterm-16color|xterm with 16 colors like aixterm,
-	ccc,
-	initc=\E]4;%p1%d;rgb\:%p2%{255}%*%{1000}%/%2.2X/%p3%{255}%*
-	      %{1000}%/%2.2X/%p4%{255}%*%{1000}%/%2.2X\E\\,
-	use=xterm+osc104, use=ibm+16color, use=xterm-new,
-
 # 256-colors is a compile-time feature of XFree86 xterm beginning with
 # xterm patch #111 (1999/7/10) -TD
 xterm+256color|xterm 256-color feature,
